lambda/src/server/main.cljs

```diff
@@ -64,7 +65,13 @@
 
 (def server (aws-express/createServer app))
 
-(def handler (fn [event context] (aws-express/proxy server event context)))
+(def slack-handler
+  (fn [event context]
+    (aws-express/proxy server event context)))
+
+(j/defn deferred-task-handler [^:js {:as event [Record] :Records} context]
```

Comment:
This destructuring implies that we only ever want the first record, correct?

Reply:
Yes. I verified that there only ever is one record